src/Lykke.Service.BlockchainWallets.Services/WalletService.cs

```diff
@@ -267,21 +269,21 @@ public async Task<(IEnumerable<WalletWithAddressExtensionDto>, string continuati
                     {
                         wallet.Address = underlyingAddress;
 
-                        finalWallets.Add(await ConvertWalletToWalletWithAddressExtensionAsync(wallet));
+                        finalWallets.Add(ConvertWalletToWalletWithAddressExtensionAsync(wallet));
                     }
                 }
                 else
                 {
-                    finalWallets.Add(await ConvertWalletToWalletWithAddressExtensionAsync(wallet));
+                    finalWallets.Add(ConvertWalletToWalletWithAddressExtensionAsync(wallet));
```

Comment:
ConvertWalletToWalletWithAddressExtensionAsync is not async anymore. naming should  be changed